beetsplug/embedart.py

```diff
@@ -117,15 +123,10 @@ def embed_item(self, item, imagepath, maxwidth=None, itempath=None,
         if compare_threshold:
             if not self.check_art_similarity(item, imagepath,
                                              compare_threshold):
-                self._log.warn(u'Image not similar; skipping.')
+                self._log.debug(u'Image not similar; skipping.')
```

Comment:
Alright, then the same would apply to the next logging statement, `media file already contained art`, wouldn't it?